### DISCUSSION TITLE
Unable to use the SigInWithOtp

### DISCUSSION THREAD

**Discussion by ElironB** [supabase/supabase#23228] [Questions] [answered]
![image](https://github.com/supabase/supabase/assets/141769879/ae3f9c1e-1995-41de-8bf7-9972bf277ce0)

For some reason when using the SignInWithOtp it inserts the email in "actor_username" instead under email it self idk what I'm doing wrong would love some help I can send logs if needed. Thank you

**Comment by GaryAustin1** [comment, marked as answer]
From discord discussion on this.  The actor_username is just something the auth audit log uses for tracking and is not relevant to the issue.  The OP is getting an `Error sending confirmation mail` error message.  This normally means a problem with the SMTP process actually sending the email to the user.
